src/main/java/com/airlines/dao/AirplaneDao.java

```diff
@@ -0,0 +1,21 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Airplane;
+
+import java.util.List;
+
+public interface AirplaneDao {
+
+    void save(Airplane airplane);
+
+    Airplane findOne(String codeName);
+
+    List<Airplane> findAll();
+
+    boolean deleteAirplane(int id);
+
+    List<Airplane> searchAirplanes(String nameCrew);
```

Comment:
crewName

Reply:
fixed in [3d2fa27](https://github.com/vitaly-lab/airlines-app/pull/6/commits/3d2fa27df397dda1795b8b92332e2576be3067c6)